AndroidSDKTests/build.gradle

```diff
@@ -50,7 +62,7 @@ dependencies {
     implementation project(':AndroidSDKPush')
     implementation project(':AndroidSDKLocation')
     implementation project(':AndroidSDKFcm')
-    implementation "androidx.appcompat:appcompat:${APPCOMPAT_LIBRARY_VERSION}"
+    implementation "androidx.appcompat:appcompat:1.2.0-beta01" // WebView in Emulator with API22 is failing with exception because of appcompat version. In 1.2.0-beta01 it is not reproduced.
```

Comment:
should we upgrade the `APPCOMPAT_LIBRARY_VERSION` directly? I've definitely had issues with mixing versions before

Reply:
Do you remember what kind of issues - runtime or compile?

Because I haven't seen any by far. Of course it is better to keep one version for all modules but I would prefer not to use beta versions in our published packages. This will cause the clients to download that dependency too and they might not be ok.

There is a task ([LP-11376](https://leanplum.atlassian.net/browse/LP-11376)) in backlog from yesterday that will address that version difference and will do it as soon as there is a release version available.